geo-data-extensions/src/main/java/org/gridsuite/geodata/extensions/Coordinate.java

```diff
@@ -26,4 +28,21 @@ public Coordinate(Coordinate coordinate) {
         this.lon = coordinate.getLon();
         this.lat = coordinate.getLat();
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
it's better to generate equals and hashcode with lombok's @EqualsAndHashCode

Reply:
done